events/models.py

```diff
@@ -0,0 +1,61 @@
+from django.db import models
+from django.contrib.gis.db import models
+
+from users.models import User
+
+
+class Category(models.Model):
+    category_name = models.CharField(max_length=255)
+    parent = models.ForeignKey(
+        "Category", 
+        related_name="categories", 
+        null=True, 
+        blank=True, 
+        on_delete=models.CASCADE
+    )
+
+
+class Event(models.Model):
+    ONSITE = 'On-site'
+    ONLINE = 'Online'
+    HYBRID = 'Hybrid'
+    
+    EVENT_TYPE_CHOICES = [
+        (ONSITE, 'On-site'),
+        (ONLINE, 'Online'),
+        (HYBRID, 'Hybrid'),
+    ]
+
+    name = models.CharField(max_length=32)
+    description = models.TextField()
+    event_type = models.CharField(
+        max_length=20,
+        choices=EVENT_TYPE_CHOICES,
+        default=ONSITE,
+    )
+    registration_start_time = models.DateTimeField()
+    registration_end_time = models.DateTimeField()
+    meeting_link = models.URLField()
+    seat_limit = models.PositiveIntegerField()
+    unique_link = models.URLField(unique=True)
```

Comment:
what is this for?

Reply:
I added this field to store the url specific for every event. Should i do it like this or should i just use the event's id ?

Reply:
shouldn't this be auto generated and not editable by user?

Reply:
Yes, you are right. I have removed this attribute.